tests/cli/ConfigExtendsCest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+class ConfigExtendsCest
+{
+    /**
+     * @param CliGuy $I
+     */
+    public function runIncludedSuites(\CliGuy $I)
+    {
+        $I->amInPath('tests/data/config_extends');
+        $I->executeCommand('run');
+
+        $I->seeInShellOutput('✔');
```

Comment:
on Windows this character is not displayed (by default). Check for something else.
See Appveyor failing build